src/modules/addDevice/addDevice.view.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import { Flex, InputItem, List, Button, WhiteSpace, WingBlank, TextareaItem, NoticeBar } from 'antd-mobile';
+import { observable, reaction, observe } from 'mobx';
+import { inject, observer } from 'mobx-react';
+import { Bind } from 'lodash-decorators';
+import DeviceService from '../../services/DeviceService';
+
+@inject('userStore')
+@observer
+class AddDevice extends Component {
+  @observable name = '';
```

Comment:
You can use one observable object eg `@observable device = new DeviceModel()`, so you don't have to do magic in line 48. That's what models are for :)